app/Model/Event.php

```diff
@@ -1293,10 +1293,14 @@ public function fetchEvent($user, $options = array()) {
 				),
 			)
 		);
-		if ($user['Role']['perm_site_admin']) {
+		if ($user['Role']['perm_site_admin'] | $user['Role']['perm_admin']) {
```

Comment:
was it intentional @cristianbell that you removed this change completely instead of fixing it?